pupil_src/shared_modules/av_writer.py

```diff
@@ -190,35 +184,39 @@ def write_video_frame(self, input_frame):
             self.container.mux(packet)
         self.current_frame_idx += 1
         self.timestamps.append(input_frame.timestamp)
-        if self.audio_export:
-            for audio_packet in self.audio_rec.demux():
-                if self.audio_packets_decoded >= len(self.audio_ts):
+        if self.audio:
+            for audio_packet in self.audio.container.demux():
+                if self.audio_packets_decoded >= len(self.audio.timestamps):
                     logger.debug(
                         "More audio frames decoded than there are timestamps: {} > {}".format(
-                            self.audio_packets_decoded, len(self.audio_ts)
+                            self.audio_packets_decoded, len(self.audio.timestamps)
                         )
                     )
                     break
                 audio_pts = int(
-                    (self.audio_ts[self.audio_packets_decoded] - self.start_time)
-                    / self.audio_export.time_base
+                    (
+                        self.audio.timestamps[self.audio_packets_decoded]
+                        - self.start_time
+                    )
+                    / self.audio_export_stream.time_base
                 )
                 audio_packet.pts = audio_pts
                 audio_packet.dts = audio_pts
-                audio_packet.stream = self.audio_export
+                audio_packet.stream = self.audio_export_stream
                 self.audio_packets_decoded += 1
 
-                if audio_pts * self.audio_export.time_base < 0:
+                if audio_pts * self.audio_export_stream.time_base < 0:
```

Comment:
I don't get what `audio_pts * self.audio_export_stream.time_base` is and you have this computation three times. Could you think of an explanatory variable name where you could store the value?